scripts/test-vector-validation/README.md

```diff
@@ -0,0 +1,20 @@
+# Test Vector Validation
+
+## Description
+
+Validates test vectors in [`web5-test-vectors`](../../web5-test-vectors/) directory. Uses [vectors.schema.json](../../web5-test-vectors/vectors.schema.json) to validate.
+
+> [!NOTE]
+> Runs automatically anytime a change is made in [`web5-test-vectors`](../../web5-test-vectors/) or to anything in this directory
+
+## Setup
+
+```bash
```

Comment:
Perhaps add instructions on the versions of node and non that need to be installed?

Reply:
done: https://github.com/TBD54566975/sdk-development/commit/b9e67a9ba77262ffd94b1d2531e7dda4891b595b